core/modules/image/tests/token.test

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * @file
+ * Tests for image.module.
+ */
+
+/**
+ * Test the Image Style Token module.
+ */
+class ImageStyleTokenTests extends BackdropWebTestCase {
+
+  /**
+   * {@inheritdoc}
+   */
+  function setUp(array $modules = array()) {
+    parent::setUp($modules);
+
+    // Create an admin user and log them in.
+    $admin_user = $this->backdropCreateUser(array(
+      'bypass node access',
+      'access content overview',
+      'administer nodes',
+    ));
```

Comment:
This is some odd indentation. I think that it should be like this instead (but unsure):

```suggestion
    $admin_user = $this->backdropCreateUser(
      array(
        'bypass node access',
        'access content overview',
        'administer nodes',
      )
    );
```

Reply:
I don't think we have standards for how to define a long array from within a function call, but the ideal would be to define the array before we call the function...  like...

```php
// Create an admin user and log them in.
    $permissions = array(
      'bypass node access',
      'access content overview',
      'administer nodes',
    );
    $admin_user = $this->backdropCreateUser($permissions);
```